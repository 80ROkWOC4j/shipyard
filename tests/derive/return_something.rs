use shipyard::*;

#[system(Test)]
fn run() -> usize { 0 }