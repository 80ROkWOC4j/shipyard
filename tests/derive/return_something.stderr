error: custom attribute panicked
 --> $DIR/return_something.rs:3:1
  |
3 | #[system(Test)]
  | ^^^^^^^^^^^^^^^
  |
  = help: message: run should not return anything.
